tensorflow/python/keras/layers/normalization_test.py

```diff
@@ -138,6 +138,27 @@ def test_batchnorm_convnet_channel_last(self):
     np.testing.assert_allclose(np.mean(out, axis=(0, 1, 2)), 0.0, atol=1e-1)
     np.testing.assert_allclose(np.std(out, axis=(0, 1, 2)), 1.0, atol=1e-1)
 
+  @keras_parameterized.run_all_keras_modes
+  def test_batchnorm_convnet_channel_last_3d_fused(self):
```

Comment:
I would also add a test with simple hand-written inputs. No need to test gradients. I'm worried this won't catch errors in `_get_shape_and_axis_for_fused`. Note gamma and beta default to ones and zeros, so you can effectively ignore them in tests that don't take gradients.

Reply:
Done.